controllers/actions.github.com/ephemeralrunner_controller.go

```diff
@@ -295,30 +294,36 @@ func (r *EphemeralRunnerReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 }
 
 func (r *EphemeralRunnerReconciler) cleanupRunnerFromService(ctx context.Context, ephemeralRunner *v1alpha1.EphemeralRunner, log logr.Logger) (ctrl.Result, error) {
-	actionsError := &actions.ActionsError{}
 	err := r.deleteRunnerFromService(ctx, ephemeralRunner, log)
-	if err != nil {
-		if errors.As(err, &actionsError) &&
-			actionsError.StatusCode == http.StatusBadRequest &&
-			strings.Contains(actionsError.ExceptionName, "JobStillRunningException") {
-			log.Info("Runner is still running the job. Re-queue in 30 seconds")
-			return ctrl.Result{RequeueAfter: 30 * time.Second}, nil
+	if err == nil { // if NO error
```

Comment:
Why do we need to flip this? It's a bit counter-intuitive to have the happy path indented 🤔

Reply:
Good point! I flipped this because error handling was huge, so I wanted to exit here to avoid having large if body (which I dislike more than not having happy path on the left :smile: ) Great point, I'll change it! Thanks!